portal/guides/dynamic-allowlist-edition.mdx

```diff
@@ -0,0 +1,819 @@
+---
+title: Create a dynamic allowlist with Airtable and Next.js
+author: Avneesh Agarwal
+authorAvatar: "/assets/portal/authors/avneesh.jpg"
+authorTwitter: "https://twitter.com/avneesh0612"
+draft: false
+tags: [edition, react, signature, mint, next.js]
+summary: "Learn how to create a dynamic allowlist where specific wallets can mint your ERC-1155 NFT and we will also see how to allow them to limit them to only 1 NFT!"
+image: "/assets/portal/guides/dynamic-allowlist-edition/thumbnail.png"
+date: "2022-04-28"
+updated: ""
+---
+
+## Introduction
+
+In this guide, we are going to create an [Edition](/contracts/edition) where people will be able to sign up their wallets to an allowlist and those wallets can later mint the NFT!
+
+Sounds cool, right!? Let's get started.
+
+## Setup
+
+I am going to use the [Next typescript starter template of thirdweb](https://github.com/thirdweb-example/next-typescript-starter) for this guide. If you already have a Next.js app you can simply follow these steps to get started:
+
+- Install `@thirdweb-dev/react` and `@thirdweb-dev/sdk`
+- Add metamask authentication to the site. You can follow this [guide](/guides/add-connectwallet-to-your-website)
+- You all are set to go now!
+
+If you are also going to use the [Next TypeScript template](https://github.com/thirdweb-example/next-typescript-starter) then follow these steps
+
+- Click on the use template button in the repo
+- Clone the repo
+- Install the dependencies:
+
+```bash
+npm install # npm
+
+yarn install # yarn
+```
+
+By default the network in `_app.tsx` is Mainnet, we need to change it to Mumbai
+
+```tsx:_app.tsx
+import type { AppProps } from 'next/app';
+import { ChainId, ThirdwebProvider } from '@thirdweb-dev/react';
+
+// This is the chainId your dApp will work on.
+const activeChainId = ChainId.Mumbai;
+
+function MyApp({ Component, pageProps }: AppProps) {
+  return (
+    <ThirdwebProvider desiredChainId={activeChainId}>
+      <Component {...pageProps} />
+    </ThirdwebProvider>
+  );
+}
+
+export default MyApp;
+```
+
+## Creating an edition and NFT
+
+We also need to create an edition and an NFT inside of the edition to be able to mint them to the users. So, go to the [thirdweb dashboard](https://thirdweb.com/dashboard) and create an edition. I am doing it with an Edition contract for this demo but it can be also done with one of one NFTs (ERC-721).
```

Comment:
```suggestion
We also need to create an edition and an NFT inside of the edition to be able to mint them to the users. So, go to the [thirdweb dashboard](https://thirdweb.com/dashboard) and create an edition. I am doing it with an [Edition](/contracts/edition) contract for this demo but it can be also done with [one of one NFTs](/contracts/nfts-and-tokens) (ERC-721).
```